app.js

```diff
@@ -0,0 +1,121 @@
+const http = require("http");
+const port = 3000;
+const server = http.createServer((req, res) => {
+    res.statusCode = 200;
+    res.setHeader("Content-Type", "application/json");
+    res.setHeader("Access-Control-Allow-Origin", "*");
+
+    const results = [
+        {
+          "gender": "female",
+          "name": {
+            "title": "mademoiselle",
+            "first": "ruth",
+            "last": "nicolas"
+          },
+          "nat": "CH"
+        },
+        {
+          "gender": "female",
+          "name": {
+            "title": "miss",
+            "first": "رها",
+            "last": "سلطانی نژاد"
+          },
+          "nat": "IR"
+        },
+        {
+          "gender": "female",
+          "name": {
+            "title": "mrs",
+            "first": "patricia",
+            "last": "hale"
+          },
+          "nat": "GB"
+        },
+        {
+          "gender": "male",
+          "name": {
+            "title": "mr",
+            "first": "fernando",
+            "last": "cooper"
+          },
+          "nat": "US"
+        },
+        {
+          "gender": "female",
+          "name": {
+            "title": "mrs",
+            "first": "یسنا",
+            "last": "صدر"
+          },
+          "nat": "IR"
+        },
+        {
+          "gender": "male",
+          "name": {
+            "title": "mr",
+            "first": "eino",
+            "last": "tuomala"
+          },
+          "nat": "FI"
+        },
+        {
+          "gender": "female",
+          "name": {
+            "title": "ms",
+            "first": "gonca",
+            "last": "özkara"
+          },
+          "nat": "TR"
+        },
+        {
+          "gender": "male",
+          "name": {
+            "title": "mr",
+            "first": "kyle",
+            "last": "castillo"
+          },
+          "nat": "US"
+        },
+        {
+          "gender": "female",
+          "name": {
+            "title": "miss",
+            "first": "olivia",
+            "last": "kumar"
+          },
+          "nat": "NZ"
+        },
+        {
+          "gender": "male",
+          "name": {
+            "title": "monsieur",
+            "first": "raymond",
+            "last": "durand"
+          },
+          "nat": "CH"
+        }
+      ],
+      info = {
+        "seed": "2cb086ce097c87ee",
+        "results": 10,
+        "page": 1,
+        "version": "1.2"
+      }
+      
+      if(req.url === "/results"){
+        res.end(JSON.stringify(results))
+
+      } else {
+       res.end("hello")
```

Comment:
Just to keep in mind: Usually this would be a 404 error since there is no page designated to this url